lib/src/_websocket_api.dart

```diff
@@ -1,4 +1,4 @@
-import 'package:web_socket_channel/web_socket_channel.dart';
+import 'imports.dart';
```

Comment:
TBH.. I'm not a fan of the uber-imports file.

the main reason is it makes it less clear to the reader which packages this particular file depends on. Perhaps flutter is different.. but with Ruby, it was often a pain to track down the side effects that libraries have on the current file.

Reply:
Since you're traumatized with Ruby, I can go back to explicitly importing all files then ✌🏻😁.